02.calendar/cal.rb

```diff
@@ -8,23 +8,20 @@
 year = (params['y'] || Date.today.year).to_i
 month = (params['m'] || Date.today.month).to_i
 
-start_day = Date.new(year, month, 1)
-end_day = Date.new(year, month, -1)
+start_date = Date.new(year, month, 1)
+end_date = Date.new(year, month, -1)
 
-calendar_of_first_weeks_spaces = '   ' * start_day.wday
-
-def format_date(day_data)
-  "#{day_data.day.to_s.rjust(2)} "
-end
+calendar_of_first_weeks_spaces = '   ' * start_date.wday
```

Comment:
```suggestion
first_week_spaces = '   ' * start_date.wday
```

このプログラムはカレンダープログラムなので、`calendar_of`を付ける必要はあまりないかな、というのと、第1週(first week)は単数なので s はいらないかなと思いました。
（first weeksだと「最初の数週間」みたいな意味になるかも？）

Reply:
ご指摘ありがとうございます。
カレンダープログラムと分かっているので、`first_week_spaces `で十分ですね。
冗長な変数名にならないように気を付けます